app/models.py

```diff
@@ -488,7 +489,7 @@ class User(Base, ModelMixin, UserMixin, PasswordOracle):
     # bitwise flags. Allow for future expansion
     flags = sa.Column(
         sa.BigInteger,
-        default=FLAG_FREE_DISABLE_CREATE_ALIAS,
+        default=FLAG_FREE_DISABLE_CREATE_ALIAS | FLAG_FREE_NEW_ALIAS_LIMIT,
```

Comment:
Should we also update the code that sets the flag when a user is created from a partner? https://github.com/simple-login/app/blob/408e3544410bdea4752ade645a95e06f6c9d5105/app/models.py#L562